admin/views/tabs/tool/wpseo-export.php

```diff
@@ -12,10 +12,14 @@
 /* translators: %1$s expands to Yoast SEO */
 $submit_button_value = sprintf( __( 'Export your %1$s settings', 'wordpress-seo' ), 'Yoast SEO' );
 
-?><p><?php
+$wpseo_export_phrase = sprintf(
 	/* translators: %1$s expands to Yoast SEO */
-	printf( __( 'Export your %1$s settings here, to import them again later or to import them on another site.', 'wordpress-seo' ), 'Yoast SEO' );
-	?></p>
+	__( 'Export your %1$s settings here, to import them again later or to import them on another site.', 'wordpress-seo' ),
+	'Yoast SEO'
+);
+?>
+
+<p><?php esc_html( $wpseo_export_phrase ); ?></p>
```

Comment:
Missing an `echo` statement here.

Reply:
Thanks for catching that. Feeling blond now ;-) PR amended.